projects/neural_correlations/EXP1-Random/NeuCorr_Exp1.ipynb

```diff
@@ -350,11 +335,54 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "print np.mean(isi)\n",
+    "print np.std(isi)\n",
+    "print np.std(isi)/np.mean(isi)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "firingRate = 8\n",
+    "# Generate spike distribution\n",
+    "spikeCount = []\n",
+    "for cell in range(np.shape(subSpikeTrains)[0]):\n",
+    "    spikeCount.append(np.count_nonzero(subSpikeTrains[cell,:]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "bins = 25\n",
+    "plt.hist(spikeCount, bins)\n",
+    "plt.xlabel(\"Spike Count\")\n",
+    "plt.ylabel(\"Number of cells\")\n",
+    "plt.savefig(\"spikesHist_TM\")\n",
+    "plt.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "firingRate = 18\n",
```

Comment:
Firing rate should be computed from the subSpikeTrains rather than set manually. The Poisson model should always match the average firing rate of the observed spike trains.
 
For example, you can use `firingRate = np.mean(subSpikeTrains, 1) * 1000`